CHANGELOG.md

```diff
@@ -1,34 +1,75 @@
 # Changelog
 
+## 0.3.0 - 2022-12-23
+
+### 🎁 Features
+
+- **✨ More robust `aria` type checking** - [SaraVieira]
+
+  `aria-sort` and `aria-orientation` now offer more robust type checking following the guidelines of MDN you can see in their pages:
+
+  - [`aria-sort`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort)
+  - [`aria-orientation`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-orientation)
+
+  [`aria-sort PR`](https://github.com/axodotdev/axohtml/pull/12)
```

Comment:
can we move this to the title so it's like `- **Title of Thing - [Author], [issue#]/[pr#] **` and just omit the issue if there is none

Reply:
pushed, gonna merge this one into changelog and fix the rest there